docker-compose.build.yml

```diff
@@ -0,0 +1,45 @@
+services:
```

Comment:
to evaluate whether all containers build locally togehter with the util/_local_build.sh